packages/eslint-config-airbnb/test/test-base.js

```diff
@@ -2,22 +2,21 @@ import fs from 'fs';
 import path from 'path';
 import test from 'tape';
 
-const base = require('../base');
+const files = {
+  base: require('../base'),
+};
 
-const files = { base };
-
-fs.readdirSync(path.join(__dirname, '../rules')).forEach((name) => {
+const rulesDir = path.join(__dirname, '../rules');
+fs.readdirSync(rulesDir).forEach((name) => {
   if (name === 'react.js' || name === 'react-a11y.js') {
     return;
   }
 
   // eslint-disable-next-line import/no-dynamic-require
-  files[name] = require(`../rules/${name}`); // eslint-disable-line global-require
+  files[name] = require(path.join(rulesDir, name));
```

Comment:
these changes seem nice (altho the eslint override comment is still needed)

Reply:
I edited @ljharb  thank you 🚀